modules/custom/cu_core/cu_core.install

```diff
@@ -426,3 +426,48 @@ function cu_core_update_7064() {
 function cu_core_update_7065() {
   secure_permissions_rebuild();
 }
+
+/**
+ * Reset affilation value to match keys from ATLAS.
+ */
+function cu_core_update_7066() {
+  // This array contains keys of the original config with values of the new config.
+  $values = array(
+    'college_of_arts_sciences' => 'arts_sciences',
+    'leeds_school_business' => 'business',
+    'college_media_communication_information' => 'cmci',
+    'school_education' => 'education',
+    'college_engineering_applied_science' => 'engineering',
+    'institutional_equity', => 'institutional_equity_compliance',
+    'school_law' => 'law',
+    'college_music' => 'law',
+    'diversity_equity_engagement' => 'odece',
+    'research_innovation' => 'research_innovation_office',
+    'strategic_relations_communication' => 'strategic_relations_comm',
+  );
+  // Check to see if the affiliation has been set already.
+  if ($affiliation = variable_get('cu_site_affiliation_options', NULL)) {
+    // Check to see if the saved affiliation matches one of the changing values.
+    if (!empty($values[$affiliation])) {
+      variable_set('cu_site_affiliation_options', $values[$affiliation]);
+    }
+  }
+}
+
+/**
+ * Set affiliation and enable affiliation.
+ */
+function cu_core_update_7067() {
```

Comment:
Need to check and see if `cu_site_affiliation_options` is already set, I think this would overwrite existing values, including custom.